### ISSUE TITLE
Xview2 Assess Building Damage

### ISSUE THREAD

**Issue by earthpulse** [earthpulse/eotdl#22]

**Comment by Schpidi** [comment]
currently uploading. size is rather 52GB than 8

**Comment by Schpidi** [comment]
I'm struggling with the upload but I keep retrying

**Comment by Schpidi** [comment]
I'm using version 2023.6.8 of the CLI now and the resume functionality works great until the end where this happens now for me:
```
eotdl-cli datasets ingest xview2_geotiff.zip xView2
Computing checksum...
2ba820756cbf6b1ba54b7afa6c85ef6a
Ingesting dataset...
52340.18/52340.18 MB: : 5235it [7:00:57,  4.82s/it]                                                                                                                                                                                                                                                                           
Checksum mismatch. Dataset deleted.
```

and for consecutive runs I get:
```
eotdl-cli datasets ingest xview2_geotiff.zip xView2
Computing checksum...
2ba820756cbf6b1ba54b7afa6c85ef6a
Ingesting dataset...
52340.18/52340.18 MB: : 5235it [00:51, 102.07it/s]                                                                                                                                                                                                                                                                            
An error occurred (NoSuchUpload) when calling the ListParts operation: The specified upload does not exist. The upload ID may be invalid, or the upload may have been aborted or completed.
```

Any advice @juansensio? Thanks

**Comment by earthpulse** [comment]
Yes, this is the same problem than before. For some reason the uploaded file is not correct foe datasets over 100GB, even though all the chunks are uploaded. Need to investigate more, also opened to ideas or suggestions. Works fine locally, getting the same error in my own cloud deployment for testing.

Juan B. Pedro Costa
👤 CTO @ EarthPulse
☎️ +34 646 66 72 84
✉️ ***@***.***
🌐 earthpulse.es

El 10 jun 2023, a las 11:28, Stephan Meißl ***@***.***> escribió:

﻿

I'm using version 2023.6.8 of the CLI now and the resume functionality works great until the end where this happens now for me:

eotdl-cli datasets ingest xview2_geotiff.zip xView2
Computing checksum...
2ba820756cbf6b1ba54b7afa6c85ef6a
Ingesting dataset...
52340.18/52340.18 MB: : 5235it [7:00:57,  4.82s/it]
Checksum mismatch. Dataset deleted.


and for consecutive runs I get:

eotdl-cli datasets ingest xview2_geotiff.zip xView2
Computing checksum...
2ba820756cbf6b1ba54b7afa6c85ef6a
Ingesting dataset...
52340.18/52340.18 MB: : 5235it [00:51, 102.07it/s]
An error occurred (NoSuchUpload) when calling the ListParts operation: The specified upload does not exist. The upload ID may be invalid, or the upload may have been aborted or completed.


Any advice @juansensio<https://github.com/juansensio>? Thanks

**Comment by Schpidi** [comment]
I see, just note, the zip I'm uploading is 'only' 52 GB so well below 100.

**Comment by earthpulse** [comment]
Ok… then the error might be happening at lower file sizes than I anticipated. I’ll continue to take a look on my side.

Thanks.

El 10 jun 2023, a las 12:07, Juan Pedro ***@***.***> escribió:

Yes, this is the same problem than before. For some reason the uploaded file is not correct foe datasets over 100GB, even though all the chunks are uploaded. Need to investigate more, also opened to ideas or suggestions. Works fine locally, getting the same error in my own cloud deployment for testing.

Juan B. Pedro Costa
👤 CTO @ EarthPulse
☎️ +34 646 66 72 84
✉️ ***@***.******@***.***>
🌐 earthpulse.es<http://earthpulse.es/>

El 10 jun 2023, a las 11:28, Stephan Meißl ***@***.***> escribió:

﻿


I'm using version 2023.6.8 of the CLI now and the resume functionality works great until the end where this happens now for me:

eotdl-cli datasets ingest xview2_geotiff.zip xView2
Computing checksum...
2ba820756cbf6b1ba54b7afa6c85ef6a
Ingesting dataset...
52340.18/52340.18 MB: : 5235it [7:00:57,  4.82s/it]
Checksum mismatch. Dataset deleted.


and for consecutive runs I get:

eotdl-cli datasets ingest xview2_geotiff.zip xView2
Computing checksum...
2ba820756cbf6b1ba54b7afa6c85ef6a
Ingesting dataset...
52340.18/52340.18 MB: : 5235it [00:51, 102.07it/s]
An error occurred (NoSuchUpload) when calling the ListParts operation: The specified upload does not exist. The upload ID may be invalid, or the upload may have been aborted or completed.


Any advice @juansensio<https://github.com/juansensio>? Thanks

**Comment by Schpidi** [comment]
I'm now using the latest develop branch and just got this sequence of outputs when running `eotdl-cli datasets ingest xview2_geotiff.zip xView2`:

```
Computing checksum...
2ba820756cbf6b1ba54b7afa6c85ef6a
Ingesting dataset...
1600.00/52340.18 MB:   3%|███████▊                                                        | 16/523 [34:01<17:58:24, 127.62s/it]
('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
```

```
Computing checksum...
2ba820756cbf6b1ba54b7afa6c85ef6a
Ingesting dataset...
1600.00/52340.18 MB:   3%|███████▊                                                          | 16/523 [02:09<1:08:13,  8.07s/it]
Checksum mismatch. Dataset deleted.
```

```
Computing checksum...
2ba820756cbf6b1ba54b7afa6c85ef6a
Ingesting dataset...
1600.00/52340.18 MB:   3%|███████▊                                                        | 16/523 [02:00<1:03:26,  7.51s/it]
An error occurred (NoSuchUpload) when calling the ListParts operation: The specified upload does not exist. The upload ID may be invalid, or the upload may have been aborted or completed.
```

What should I do now?

Cc @achtsnits

**Comment by juansensio** [comment]
Hi, I just uploaded a new version and was able to upload the dataset. Closing the issue.
